src/test/java/uk/gov/companieshouse/confirmationstatementapi/configuration/InterceptorConfigRouteMatchingTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.companieshouse.confirmationstatementapi.configuration;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.servlet.HandlerExecutionChain;
+import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest()
```

Comment:
trivia - annotation looks a bit odd with the empty ()

Reply:
Fixed.